config/settings-production.yml

```diff
@@ -42,13 +42,25 @@ buffy:
       remove_labels:
         - "0/pre-review-checks"
         - "New Submission!"
+      add_as_assignee: true
     remove_editor:
       only: editors
-      no_editor_text: "To be decided"
     reviewers_list:
       only: editors
       sample_value: "@reviewer-login"
       add_as_assignee: false
+    add_remove_checklist:
```

Comment:
i'm assuming that similar to the label responded, this responder also needs an array rather than multiple instances of a responder call to add a checklist.